rust_snuba/rust_arroyo/src/processing/mod.rs

```diff
@@ -201,7 +202,7 @@ impl<'a, TPayload: 'static + Clone> StreamProcessor<'a, TPayload> {
     }
 
     pub fn signal_shutdown(&mut self) {
```

Comment:
```suggestion
    pub fn signal_shutdown(&self) {
```

then you should be able to use it

Reply:
oh now i see what's goign on. ok. should we remove this method?

Reply:
Updated to `ProcessorHandle.signal_shutdown()` as discussed offline